### PR TITLE
resource_groups: add description about default resource group

### PR DESCRIPTION
**Pull request by nolouch**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

https://github.com/pingcap/tidb/pull/42596

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: information-schema/information-schema-resource-groups.md [hunk 1]
```diff
@@ -52,6 +59,10 @@ mysql> SELECT * FROM information_schema.resource_groups WHERE NAME = 'rg1';
 `RESOURCE_GROUPS` 表中列的含义如下：
 
 * `NAME`：资源组名称。
-* `RU_PER_SEC`：资源组的回填速度，单位为每秒回填的 [Request Unit (RU)](/tidb-resource-control.md#什么是-request-unit-ru) 数量。
+* `RU_PER_SEC`：资源组的回填速度，单位为每秒回填的 [Request Unit (RU)](/tidb-resource-control.md#什么是-request-unit-ru) 数量, 取值范围为 (0, MAXINT32]。
```
[the remark this answers is not in the document]
Comment (nolouch) [reply information-schema/information-schema-resource-groups.md]: 这个取值范围还没有限制，后面在了限制再说明。

File: sql-statements/sql-statement-alter-resource-group.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ TiDB 支持以下 `DirectResourceGroupOption`, 其中 [Request Unit (RU)](/tidb-
 
 | 参数            | 含义           | 举例                                   |
 |---------------|--------------|--------------------------------------|
-| `RU_PER_SEC`  | 每秒 RU 填充的速度 | `RU_PER_SEC = 500` 表示此资源组每秒回填 500 个 RU。 |
+| `RU_PER_SEC`  | 每秒 RU 填充的速度 | `RU_PER_SEC = 500` 表示此资源组每秒回填 500 个 RU。 取值范围为 (0, MAXINT32]，默认值为 `MAXINT32` (即 `UNLIMITED`)。 |
```
Comment (hfxsd) [inline sql-statements/sql-statement-alter-resource-group.md:42]:
```suggestion
| `RU_PER_SEC`  | 每秒 RU 填充的速度 | `RU_PER_SEC = 500` 表示此资源组每秒回填 500 个 RU。 取值范围为 (0, MAXINT32]，默认值为 `UNLIMITED` (等同于 `MAXINT32`)。 |
```

File: information-schema/information-schema-resource-groups.md [hunk 3]
```diff
@@ -55,3 +82,7 @@ mysql> SELECT * FROM information_schema.resource_groups WHERE NAME = 'rg1';
 * `RU_PER_SEC`：资源组的回填速度，单位为每秒回填的 [Request Unit (RU)](/tidb-resource-control.md#什么是-request-unit-ru) 数量。
 * `PRIORITY`：任务在 TiKV 上处理的绝对优先级。不同的资源按照 `PRIORITY` 的设置进行调度，`PRIORITY` 高的任务会被优先调度。如果资源组的 `PRIORITY` 相同，则会根据 `RU_PER_SEC` 的配置按比例调度。如果不指定 `PRIORITY`，资源组的默认优先级为 `MEDIUM`。
 * `BURSTABLE`：是否允许此资源组超额使用剩余的系统资源。
+
+> **注意：**
+>
+> TiDB 默认预置了 `default` 资源组。`default` 资源组是所有语句执行默认使用的资源组。默认配置 `RU_PER_SEC` 为 `UNLIMITED` (等同于`MaxInt32=2147483647`), 且为 `BURSTABLE` 模式。在新建配置其他资源组时，建议根据实际情况修改 `default` 资源组的配置。
```
Comment (glorv) [inline information-schema/information-schema-resource-groups.md:88]: `MaxInt32` 是一个编程语言相关的数据，放在文档里面这样描述感觉不太合适？就直接统一用 `2147483647` 如何？

Comment (nolouch) [reply information-schema/information-schema-resource-groups.md]: 改成 INT 说明一下了，INT 是 mysql 里面的类型。

**Review by glorv — COMMENTED** [review]
LGTM

File: information-schema/information-schema-resource-groups.md [hunk 4]
```diff
@@ -55,3 +82,7 @@ mysql> SELECT * FROM information_schema.resource_groups WHERE NAME = 'rg1';
 * `RU_PER_SEC`：资源组的回填速度，单位为每秒回填的 [Request Unit (RU)](/tidb-resource-control.md#什么是-request-unit-ru) 数量。
 * `PRIORITY`：任务在 TiKV 上处理的绝对优先级。不同的资源按照 `PRIORITY` 的设置进行调度，`PRIORITY` 高的任务会被优先调度。如果资源组的 `PRIORITY` 相同，则会根据 `RU_PER_SEC` 的配置按比例调度。如果不指定 `PRIORITY`，资源组的默认优先级为 `MEDIUM`。
 * `BURSTABLE`：是否允许此资源组超额使用剩余的系统资源。
+
+> **注意：**
+>
+> TiDB 默认预置了 `default` 资源组。`default` 资源组是所有语句执行默认使用的资源组。默认配置 `RU_PER_SEC` 为 `UNLIMITED` (等同于`INT`类型最大值，即 `2147483647`), 且为 `BURSTABLE` 模式。在新建配置其他资源组时，建议根据实际情况修改 `default` 资源组的配置。
```
Comment (qiancai) [inline information-schema/information-schema-resource-groups.md]:
```suggestion
> TiDB 集群在初始化时会自动创建 `default` 资源组，其 `RU_PER_SEC` 的默认值为 `UNLIMITED` (等同于`INT`类型最大值，即 `2147483647`)，且为 `BURSTABLE` 模式。所有 SQL 语句的执行默认使用 `default` 资源组。在新建配置其他资源组时，建议根据实际情况修改 `default` 资源组的配置。
```

Comment (qiancai) [reply information-schema/information-schema-resource-groups.md]: reason: 保持和 sql-statements/sql-statement-create-resource-group.md 中对 default 组的描述一致

File: sql-statements/sql-statement-create-resource-group.md [hunk 5]
```diff
@@ -48,7 +48,7 @@ TiDB 支持以下 `DirectResourceGroupOption`, 其中 [Request Unit (RU)](/tidb-
 > **注意：**
 >
 > - `CREATE RESOURCE GROUP` 语句只能在全局变量 [`tidb_enable_resource_control`](/system-variables.md#tidb_enable_resource_control-从-v660-版本开始引入) 设置为 `ON` 时才能执行。
-> - TiDB 集群在初始化时会自动创建 `default` 资源组，所有未绑定资源组的请求都将自动绑定至此资源组。
+> - TiDB 集群在初始化时会自动创建 `default` 资源组，其 `RU_PER_SEC` 的默认值为 `UNLIMITED` (等同于`INT`类型最大值，即 `2147483647`), 且为 `BURSTABLE` 模式。所有未绑定资源组的请求都将自动绑定至此资源组。在新建配置其他资源组时，建议根据实际情况修改 `default` 资源组的配置。
```
Comment (hfxsd) [inline sql-statements/sql-statement-create-resource-group.md:51]:
```suggestion
> - TiDB 集群在初始化时会自动创建 `default` 资源组，其 `RU_PER_SEC` 的默认值为 `UNLIMITED` (等同于`INT`类型最大值，即 `2147483647`)，且为 `BURSTABLE` 模式。所有未绑定资源组的请求都将自动绑定至此资源组。在新建配置其他资源组时，建议根据实际情况修改 `default` 资源组的配置。
```

File: tidb-resource-control.md [hunk 6]
```diff
@@ -125,7 +125,7 @@ ALTER USER usr2 RESOURCE GROUP rg2;
 > **注意：**
 >
 > - 使用 `CREATE USER` 或者 `ALTER USER` 将用户绑定到资源组后，只会对该用户新建的会话生效，不会对该用户已有的会话生效。
-> - TiDB 集群在初始化时会自动创建 `default` 资源组，对于没有绑定资源组的语句会自动绑定至此资源组。此资源组不支持删除，但允许修改其 RU 的配置。
+> - TiDB 集群在初始化时会自动创建 `default` 资源组，其 `RU_PER_SEC` 的默认值为 `UNLIMITED` (等同于`INT`类型最大值，即 `2147483647`), 且为 `BURSTABLE` 模式。对于没有绑定资源组的语句会自动绑定至此资源组。此资源组不支持删除，但允许修改其 RU 的配置。
```
Comment (hfxsd) [inline tidb-resource-control.md:128]:
```suggestion
> - TiDB 集群在初始化时会自动创建 `default` 资源组，其 `RU_PER_SEC` 的默认值为 `UNLIMITED` (等同于`INT`类型最大值，即 `2147483647`)，且为 `BURSTABLE` 模式。对于没有绑定资源组的语句会自动绑定至此资源组。此资源组不支持删除，但允许修改其 RU 的配置。
```

File: sql-statements/sql-statement-create-resource-group.md (hunk 5 above)
Comment (qiancai) [inline sql-statements/sql-statement-create-resource-group.md]:
```suggestion
> - TiDB 集群在初始化时会自动创建 `default` 资源组，其 `RU_PER_SEC` 的默认值为 `UNLIMITED` (等同于`INT`类型最大值，即 `2147483647`)，且为 `BURSTABLE` 模式。所有未绑定资源组的请求都将自动绑定至此资源组。在新建配置其他资源组时，建议根据实际情况修改 `default` 资源组的配置。
```

File: information-schema/information-schema-resource-groups.md [hunk 7]
```diff
@@ -55,3 +82,7 @@ mysql> SELECT * FROM information_schema.resource_groups WHERE NAME = 'rg1';
 * `RU_PER_SEC`：资源组的回填速度，单位为每秒回填的 [Request Unit (RU)](/tidb-resource-control.md#什么是-request-unit-ru) 数量。
 * `PRIORITY`：任务在 TiKV 上处理的绝对优先级。不同的资源按照 `PRIORITY` 的设置进行调度，`PRIORITY` 高的任务会被优先调度。如果资源组的 `PRIORITY` 相同，则会根据 `RU_PER_SEC` 的配置按比例调度。如果不指定 `PRIORITY`，资源组的默认优先级为 `MEDIUM`。
 * `BURSTABLE`：是否允许此资源组超额使用剩余的系统资源。
+
+> **注意：**
+>
+> TiDB 默认预置了 `default` 资源组。`default` 资源组是所有语句执行默认使用的资源组。其 `RU_PER_SEC` 的默认值为 `UNLIMITED` (等同于`INT`类型最大值，即 `2147483647`)，且为 `BURSTABLE` 模式。在新建配置其他资源组时，建议根据实际情况修改 `default` 资源组的配置。
```
Comment (qiancai) [inline information-schema/information-schema-resource-groups.md:88]: 这里要写 “所有未绑定资源组的请求都将自动绑定至此资源组。” 吗

File: information-schema/information-schema-resource-groups.md [hunk 8]
```diff
@@ -31,16 +31,43 @@ DESC resource_groups;
...
 | Resource_Group | Create Resource Group                                         |
 +----------------+---------------------------------------------------------------+
 | rg1            | CREATE RESOURCE GROUP `rg1` RU_PER_SEC=1000 PRIORITY="MEDIUM" |
 +----------------+---------------------------------------------------------------+
 1 row in set (0.00 sec)
-mysql> SELECT * FROM information_schema.resource_groups WHERE NAME = 'rg1';
+```
+
+```sql
+SELECT * FROM information_schema.resource_groups WHERE NAME = 'rg1';
```
Comment (hfxsd) [inline information-schema/information-schema-resource-groups.md:67]:
```suggestion
SELECT * FROM information_schema.resource_groups WHERE NAME = 'rg1'; -- 查看资源组 rg1
```

File: information-schema/information-schema-resource-groups.md (hunk 7 above)
Comment (qiancai) [reply information-schema/information-schema-resource-groups.md:88]:
```suggestion
> TiDB 集群在初始化时会自动创建 `default` 资源组，其 `RU_PER_SEC` 的默认值为 `UNLIMITED` (等同于`INT`类型最大值，即 `2147483647`)，且为 `BURSTABLE` 模式。所有未绑定资源组的请求都将自动绑定至此资源组。在新建配置其他资源组时，建议根据实际情况修改 `default` 资源组的配置。
```

File: information-schema/information-schema-resource-groups.md [hunk 9]
```diff
@@ -31,16 +31,43 @@ DESC resource_groups;
...
+```sql
++---------+------------+----------+-----------+
+| NAME    | RU_PER_SEC | PRIORITY | BURSTABLE |
++---------+------------+----------+-----------+
+| default | UNLIMITED  | MEDIUM   | YES       |
++---------+------------+----------+-----------+
+```
+
+```sql
+CREATE RESOURCE GROUP rg1 RU_PER_SEC=1000; -- 创建资源组 rg1
```
Comment (hfxsd) [inline information-schema/information-schema-resource-groups.md:46]:
```suggestion
CREATE RESOURCE GROUP rg1 RU_PER_SEC=1000; -- 创建资源组 `rg1`
```

File: information-schema/information-schema-resource-groups.md [hunk 10]
```diff
@@ -31,16 +31,43 @@ DESC resource_groups;
...
+```
+
+```sql
 Query OK, 0 rows affected (0.34 sec)
-mysql> SHOW CREATE RESOURCE GROUP rg1; -- 显示 rg1 资源组的定义
+```
+
+```sql
+SHOW CREATE RESOURCE GROUP rg1; -- 显示 rg1 资源组的定义
```
Comment (hfxsd) [inline information-schema/information-schema-resource-groups.md:54]:
```suggestion
SHOW CREATE RESOURCE GROUP rg1; -- 显示 `rg1` 资源组的定义
```

**Review by hfxsd — APPROVED** [review]

**Review by qiancai — APPROVED** [review]
other LGTM

File: information-schema/information-schema-resource-groups.md [hunk 11]
```diff
@@ -55,3 +82,7 @@ mysql> SELECT * FROM information_schema.resource_groups WHERE NAME = 'rg1';
 * `RU_PER_SEC`：资源组的回填速度，单位为每秒回填的 [Request Unit (RU)](/tidb-resource-control.md#什么是-request-unit-ru) 数量。
 * `PRIORITY`：任务在 TiKV 上处理的绝对优先级。不同的资源按照 `PRIORITY` 的设置进行调度，`PRIORITY` 高的任务会被优先调度。如果资源组的 `PRIORITY` 相同，则会根据 `RU_PER_SEC` 的配置按比例调度。如果不指定 `PRIORITY`，资源组的默认优先级为 `MEDIUM`。
 * `BURSTABLE`：是否允许此资源组超额使用剩余的系统资源。
+
+> **注意：**
+>
+> TiDB 集群在初始化时会自动创建 `default` 资源组，其 `RU_PER_SEC` 的默认值为 `UNLIMITED` (等同于`INT`类型最大值，即 `2147483647`)，且为 `BURSTABLE` 模式。所有未绑定资源组的请求都将自动绑定至此资源组。在新建配置其他资源组时，建议根据实际情况修改 `default` 资源组的配置。
```
Comment (qiancai) [inline information-schema/information-schema-resource-groups.md:88]:
```suggestion
> TiDB 集群在初始化时会自动创建 `default` 资源组，其 `RU_PER_SEC` 的默认值为 `UNLIMITED` (等同于 `INT` 类型最大值，即 `2147483647`)，且为 `BURSTABLE` 模式。所有未绑定资源组的请求都将自动绑定至此资源组。在新建配置其他资源组时，建议根据实际情况修改 `default` 资源组的配置。
```

**Review by songrijie — APPROVED** [review]
LGTM

**Pull request merged**